browser.js

```diff
@@ -0,0 +1,20 @@
+/** @module  audio-speaker/browser */
+'use strict'
+
+var objectAssign = require('object-assign')
+var createContext = require('audio-context')
+var createWriter = require('web-audio-write')
+
+module.exports = function createSpeaker (opts) {
+  var options = {}
+
+  options = objectAssign({
+    channels: 1,
```

Comment:
Fair point, this is the same in node and in the readme. So if you'd like to change that feel free. 😄